docs/platform/tutorials/image-processing/index.md

```diff
@@ -1,42 +1,68 @@
 # Real-time image processing
 
-In this tutorial you learn how to build a real-time image processing pipeline in Quix, using the Transport for London (TfL) traffic cameras, known as Jam Cams, the webcam on your laptop or phone,  and a [YOLO v3](https://viso.ai/deep-learning/yolov3-overview/) machine learning model. 
+In this tutorial you learn about a real-time image processing pipeline, using a [Quix template project](https://github.com/quixio/computer-vision-demo){target=_blank}. 
 
-You'll use prebuilt Code Samples to build the pipeline. A prebuilt UI is also provided that shows you where the recognized objects are located around London.
+The pipeline uses the Transport for London (TfL) traffic cameras, known as Jam Cams, as the video input. The [YOLO v8](https://docs.ultralytics.com/) machine learning model is used to identify various objects such as types of vehicles. Additional services count the vehicles and finally the data is displayed on a map which is part of the web UI that has been creatde specially for this project. 
 
-The following screenshot shows the pipeline you build in this tutorial:
+You'll fork the complete project from GitHub, and then create a Quix project from the forked repo, so you have a copy of the full pipeline code running in your Quix account. You then examine the data flow through the pipeline, using tools provided by the Quix Portal.
 
-![pipeline overview](./images/pipeline-overview.png)
+## Technologies used
 
+This template project uses the following technologies:
```

Comment:
This list feels a bit long to me. I don't think we need to mention some of the more obvious super general developer technologies like Git and S3 and we could include all the Python libraries on one line next to the mention of Python 3. Also Angular and Typescript are basically one iin the same and can be mentioned on one line.

Reply:
Also, Apache Kafka is the big missing elephant in the room!

Reply:
Yes, I've shortened and grouped. It was a bit overwhelming! :)